tools/ci_build/build.py

```diff
@@ -2810,7 +2814,7 @@ def main():
             run_subprocess([emsdk_file, "activate", emsdk_version], cwd=emsdk_dir)
 
         if args.enable_pybind and is_windows():
-            install_python_deps(args.numpy_version)
+            install_python_deps('requirements-pybind.txt',args.numpy_version)
```

Comment:
Sorry, it has to be "requirements.txt". You may put the file in a different folder, with this name.